training_config/rc/transformer_qa.jsonnet

```diff
@@ -1,38 +1,40 @@
-local transformer_model = 'bert-base-cased';
+local transformer_model = "roberta-large";
 
-local epochs = 3;
-local batch_size = 8;
+local epochs = 5;
+local batch_size = 16;
+local length_limit = 512;
+
+local seed = 100;
 
 {
   "dataset_reader": {
-      "type": "transformer_squad",
-      "transformer_model_name": transformer_model,
-      "skip_invalid_examples": true,
-      //"max_instances": 200  // debug setting
+    "type": "transformer_squad",
+    "transformer_model_name": transformer_model,
+    "length_limit": length_limit,
+    // "max_instances": 1000,  // debug setting
   },
-  "validation_dataset_reader": self.dataset_reader + {
-      "skip_invalid_examples": false,
+  "train_data_path": "https://allennlp.s3.amazonaws.com/datasets/squad/squad-train-v2.0.json",
+  "validation_data_path": "https://allennlp.s3.amazonaws.com/datasets/squad/squad-dev-v2.0.json",
```

Comment:
How does this pass the test if we can't substitute the inputs?

Reply:
`max_instances` is overridden and set to a small number in the tests, so the substitution is really only necessary for datasets that are not public.